codegen/types.go

```diff
@@ -143,7 +143,7 @@ func goTypeDefObject(o design.Object) string {
 	var ss []string
 	ss = append(ss, "struct {")
 	o.WalkAttributes(func(name string, at *design.AttributeExpr) error {
-		ss = append(ss, fmt.Sprintf("%s %s", name, GoTypeName(at.Type)))
+		ss = append(ss, fmt.Sprintf("\t%s %s", name, GoTypeName(at.Type)))
```

Comment:
Do you mean should this returns like

```go
struct {
IntField int
StringField string
}
```

instead of

```go
struct {
	IntField int
	StringField string
}
```

?